<?xml version="1.0"?>
<launch>
    <node name="predictor_node" pkg="detector_2d" type="predictor.py" output="screen">
        <rosparam command="load" file="$(find detector_2d)/config/main_cfg.yaml"/>
    </node>

</launch>
